Cargo.toml

```diff
@@ -21,6 +21,9 @@ travis-ci = { repository = "alteous/gltf" }
 gltf-json = "0.1"
 lazy_static = "0.2"
 
+[patch.crates-io]
```

Comment:
I think one can specify both `path` and `version` in the manifest format and `cargo` will choose accordingly.